src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperOutput.cs

```diff
@@ -96,7 +83,7 @@ public bool ContentSet
         {
```

Comment:
@NTaylorMullen disagree.  we've avoided that C# 6.0 feature (because it's ugly and adds nothing over what @sornaks has here)